Maxis/Interfaces/CustomPrincipal.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Principal;
+using System.Web;
+
+namespace Maxis.Interfaces
+{
+    public class CustomPrincipal : ICustomPrincipal
```

Comment:
inherit Iproncipal to customprincipal class and remove Icustomprincipal interface